src/components/ConversionServerErrorModal/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from "antd";
+import React from "react";
+
+import CustomModal from "../CustomModal";
+import { Exclamation } from "../Icons";
+
+import styles from "./style.css";
+
+interface ConversionServerErrorModalProps {
+    closeModal: () => void;
+}
+
+const ConversionServerErrorModal: React.FC<ConversionServerErrorModalProps> = ({
+    closeModal,
+}) => {
+    return (
+        <CustomModal
+            className={styles.serverErrorModal}
+            title="File import cannot be completed"
+            open
+            footer={
+                <Button type="primary" onClick={closeModal}>
+                    OK
+                </Button>
+            }
+            centered
+            onCancel={closeModal}
+        >
+            <p className={styles.warningText}>
+                {" "}
```

Comment:
Can you delete these and try to figure out how to keep them from happening? Are you adding spaces while you're typing that then get converted?

Reply:
I've removed these and made a note for myself to look through the repo for these things and keep them from showing up.